mkosi.uki-profiles/95-emergency.conf

```diff
@@ -0,0 +1,11 @@
+# SPDX-License-Identifier: LGPL-2.1-or-later
+
+[UKIProfile]
+Profile=
+        ID=debug
+        TITLE=Boot into Emergency Mode
+
+Cmdline=
+        systemd.unit=emergency.target
+
+SignExpectedPcr=yes
```

Comment:
I don't think this should have signed PCRs? Isn't that unsafe as you get effectively a root shell in the initrd? I guess you still need a password so maybe it's fine.

Reply:
this gives you a root shell after the initrd, not within it (i.e. this uses systemd.unit=, not rd.systemd.unit=), and only after auth. 

i was thinking about this, but I think emergency mode should be something we can "recover" from, i.e. something that shouldn't corrupt the PCR state for us. because if it did that it would be much much less useful...

the alternative would be that we start measuring some special pcr phase from both rescue and emergency mode, but then sign the expected pcrs for both of those cases, too. but not sure what the benefit would be.